pkg/commands/app.go

```diff
@@ -195,13 +197,15 @@ func NewRootCommand(globalFlags *flag.GlobalFlagGroup) *cobra.Command {
 				return xerrors.Errorf("flag bind error: %w", err)
 			}
 
+			log.InitLogger(true, false)
```

Comment:
I'm sorry, but this is not what I meant. I was supposed to describe the details after this PR. Now, even if `--debug` is not specified, Trivy always shows debug messages, which confuses users.

```
$ trivy image alpine:3.20
2024-07-18T12:56:11+04:00       DEBUG   Default config file "file_path=trivy.yaml" not found, using built in values
2024-07-18T12:56:11+04:00       DEBUG   Cache dir       dir="/Users/teppei/Library/Caches/trivy"
```

We can fix it in another PR. Could you revert?

Reply:
Sorry, it was a test that I didn't meant to put in this PR.